Sources/GryphonLib/Utilities.swift

```diff
@@ -308,3 +308,21 @@ extension List {
 		return result
 	}
 }
+
+extension String {
+	func isArrayDeclaration() -> Bool {
+		if self.starts(with: "[") {
```

Comment:
If you choose to move this code to a translated file, I think you'll also have to change this method call.

Gryphon doesn't support translating `String.starts(with:)` yet, so this will cause a compilation error in Kotlin. You can either change this to `String.hasPrefix(_:)` or add a translation for `String.starts(with:)` in AuxiliaryFileContents.swift ([around here](https://github.com/vinivendra/Gryphon/blob/3601d30204875796f04aa55a7d9263d8bf00f0dc/Sources/GryphonLib/AuxiliaryFileContents.swift#L1476)).

Reply:
for now, I have moved the extension to SharedUtilities.swift file. the bootstrap tests are passing now.
If any further changes are required, most welcome.